syncapi/storage/postgres/memberships_table.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2017-2018 New Vector Ltd
+// Copyright 2019-2020 The Matrix.org Foundation C.I.C.
```

Comment:
Lies

Reply:
How is it I *never* remember to fix these?